README.md

```diff
@@ -324,7 +324,7 @@ If you think you know all about stock corporations, you can
 - 🄳 A **company** is a legal entity formed under corporate law for the purpose of conducting
   trade. In the United States,
   [several kinds of business entities](https://en.wikipedia.org/wiki/Types_of_business_entity#United_States)
-  may be formed, or [**incorporated**]\(<https://en.wikipedia.org/wiki/Incorporation_(business)>,
+  may be formed, or [**incorporated**](<https://en.wikipedia.org/wiki/Incorporation_(business)>,
```

Comment:
The URL inside the parentheses includes angle brackets (perhaps because it was copied from somewhere), which is incorrect. It's also missing the closing paren. See here for a correct example:
https://github.com/jlevy/og-equity-compensation/blame/e2.0/README.md#L452

It looks like we can get away with the parens _not_ being percent encoded (%28 or %29) as long as the paren from the URL is followed by the paren from the Markdown syntax.

Reply:
Figured out why this was happening: With two closing parens in a Wiki link, Atom Preview shows an extra paren, it does not get absorbed as part of the link. After double checking your example, did not delete the second parens in most recent commit and it looks good in GitHub, even tho the extra parens stands in Preview.

Reply:
Ah yes. The extra paren in Preview can be ignored; it's a bug.